src/cloud/live/sens-data-over.md

```diff
@@ -45,6 +45,8 @@ Because {{site.data.var.ece}} supports only the Magento production and maintenan
 
 Sensitive values are _not_ stored in the `app/etc/config.php` file. Any sensitive configurations export to the `app/etc/config.php` file during the `config:dump` process. We recommend using environment variables to store sensitive values. For an example, see [Add Magento authentication keys][auth].
 
+You can also set sensitive values using the Magento CLI command `bin/magento config:sensitive:set`. See [Sensitive or system-specific settings].
```

Comment:
```suggestion
Also, you can use the Magento CLI `bin/magento config:sensitive:set` command to set sensitive values. See [Sensitive or system-specific settings] in the _Configuration Guide_.
```

Reply:
@hguthrie  - I don't like the rewording for the first part (for me the 'noun' is "Magento CLI command" so it gets split) - it doesn't scan well for me. But happy to add the "..in the _Configuration Guide_". What do you say?

Reply:
Ending a sentence with a formatted command can be confusing because of the period. Some people will add the period in the command and then wonder why it doesn't work.

If one must end a sentence with a command as such, I suggest terminating with a colon and then the command...no period.

```
You can set sensitive values using the Magento CLI command: `bin/magento config:sensitive:set`
```

Reply:
That's a good point about the period. How about -
You can also set sensitive values using the Magento CLI command : `bin/magento config:sensitive:set`,  see [Sensitive or system-specific settings].